### PR TITLE
adding deploy external image docs

### PR DESCRIPTION
**Pull request by emanuel-quix**

### CODE REVIEW

File: docs/deploy/deploy-external-image.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Deploy an External Image
+
+This section of the documentation provides guidance on how to ***deploy an external image*** within the pipeline.
+
+Deploying an external image directly allows you to bypass application and version configurations, offering flexibility for non-standardized services or third-party containers. The deployment process is straightforward and integrated into the UI.
+
+To summarize, your workflow for deploying an external image is:
+
+1. **Prepare the external image**: Ensure it's built and pushed to a container registry accessible by the platform.
+\
```
Comment (luisquix) [inline docs/deploy/deploy-external-image.md]: typo?

File: docs/deploy/deploy-external-image.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+| Dialog Item          | Description                                                                 |
+|----------------------|-----------------------------------------------------------------------------|
+| **External Image**   | The full external image reference, including registry, name, and tag.      |
+| **Environment Variables** | Any environment variables your container needs can be specified here.      |
+| **Deployment Settings**  | Configure whether the image runs as a service or a job. Also, set CPU, memory, and replicas for scaling. |
+| **Public Access**    | Configure whether the container is accessible externally via a public URL. |
+| **State Management** | If state is enabled, a `state` folder is created to persist data across restarts. See [state management docs](./state-management.md). |
+| **Deployment Name**  | The name of the deployment. You can change it to something descriptive.    |
+
```
Comment (luisquix) [inline docs/deploy/deploy-external-image.md:50]: do we need a Yaml example?

**Review by luisquix — CHANGES_REQUESTED** [review]

File: docs/deploy/deploy-external-image.md (hunk 1 above)
Comment (emanuel-quix) [reply docs/deploy/deploy-external-image.md]: Indeed, thanks!

File: docs/deploy/deploy-external-image.md (hunk 2 above)
Comment (emanuel-quix) [reply docs/deploy/deploy-external-image.md:50]: I am pointing to the yaml configuration here https://github.com/quixio/quix-docs/blob/ac6a84447f3f3fc5995363811e4dc53e58082a2b/docs/deploy/deploy-external-image.md?plain=1#L53

**Review by luisquix — APPROVED** [review]

**Pull request merged**